docs/breaking/registration.md

```diff
@@ -0,0 +1,48 @@
+# Project Registration Update
+Sphinx is updating the way that you register projects. Instead of specifying your Gnosis Safe owners, threshold and salt nonce in your Solidity scripts, you will now register your projects via the Sphinx UI. In your script, you will simply specify the name of your project.
```

Comment:
I was planning to just message everyone directly since that's what we've been doing.

We could do something better where we detect if their script is out of date and prompt them to upgrade, but that's non-trivial, especially if we want to remove the old configuration fields from the script with this change (which is preferable IMO). Since we still don't have many users, I wouldn't bother with it.

That being said, it would be a good idea to keep in mind that integrations may be outdated when following up with users.